crates/ruma-client-api/src/message/get_message_events.rs

```diff
@@ -56,12 +66,12 @@ pub mod v3 {
 
             /// The maximum number of events to return.
             ///
-            /// Default: 10.
+            /// Default: `10`.
             #[ruma_api(query)]
             #[serde(default = "default_limit", skip_serializing_if = "is_default_limit")]
             pub limit: UInt,
 
-            /// A RoomEventFilter to filter returned events with.
+            /// A `RoomEventFilter` to filter returned events with.
```

Comment:
```suggestion
            /// A [`RoomEventFilter`] to filter returned events with.
```

Reply:
The page has the link right below, not sure whether it's worth to add another one.

Reply:
But I guess it doesn't hurt either.